lib/data_update_scripts/20221220131148_add_multiple_reactions_feature_flag.rb

```diff
@@ -0,0 +1,7 @@
+module DataUpdateScripts
+  class AddMultipleReactionsFeatureFlag
+    def run
+      FeatureFlag.add :multiple_reactions
```

Comment:
I'm curious to know if we need to add a DUS (Data Update Script) for this, which then would potentially live in our codebase and need another DUS to remove it or could we just manually add the feature flag wherever we're testing it?

Reply:
For what it's worth, this is recommended in [our documentation](https://developers.forem.com/technical-overview/feature-flags), but I agree with you that it isn't always necessary. I also agree with you that the long-term "stickiness" of data-update scripts make this better suited to longer-running feature flags than temporary roll-out flags. 

The main advantage that I see in adding a feature flag this way, is that it avoids having potentially confusing typos (was it an underscore or hash? plural or singular?). 

Anyway, I'm happy to remove it if that's preferred.